package.json

```diff
@@ -195,7 +195,7 @@
     "purgecss": "^1.3.0",
     "purgecss-whitelister": "^2.4.0",
     "purify-css": "^1.2.5",
-    "query-string": "^6.9.0",
+    "query-string": "5.0.0",
```

Comment:
Need to downgrade `query-string` to v5 because v6 only targets the latest version of Chrome, Firefox, and Safari.

query-string installation guide:
"This module targets Node.js 6 or later and the latest version of Chrome, Firefox, and Safari. If you want support for older browsers..."
https://github.com/sindresorhus/query-string#install
https://github.com/sindresorhus/query-string/issues/189

Reply:
Can we not just get rid of the lib and use URLSearchParams - not supported in IE but babel will just polyfill it